apps/web/src/actions/user/setupAction.ts

```diff
@@ -35,14 +38,60 @@ export const setupAction = errorHandlingProcedure
         companyName: z
           .string()
           .min(1, { message: 'Workspace name is a required field' }),
+        invitationToken: z.string().optional(),
       })
     },
     { type: 'formData' },
   )
   .handler(async ({ input }) => {
-    const result = await setupService(input)
+    let validatedInvitationEmail: string | undefined = undefined
+
+    if (env.INVITE_ONLY) {
```

Comment:
In this page just throw an error if `env.INVITE_ONLY`. The idea is no one do the setup without invitation no?

Let's try to keep as clean as possible default setup